assets/javascripts/discourse/controllers/admin-ticketing.js.es6

```diff
@@ -1,8 +1,48 @@
+import { observes } from 'ember-addons/ember-computed-decorators';
+
 export default Ember.Controller.extend({
   showDashboard: true,
 
+  sorter(a, b, orderKey) {
+    const priorityMap = {
+        immediate: 0,
+        urgent: 1,
+        high: 2,
+        normal: 3,
+        low: 4
+    }
+    var aValue = 0;
```

Comment:
If you wanted, you could replace these `var`s with `let` to make it feel more es6-ish, but totally not necessary